test/kpod.bats

```diff
@@ -16,34 +17,103 @@ KPOD_OPTIONS="--root $ROOT --runroot $RUNROOT --storage-driver vfs"
 	[ "$status" -eq 0 ]
+	run ${KPOD_BINARY} $KPOD_OPTIONS rmi debian:6.0.10
+	[ "$status" -eq 0 ]
 }
 
 @test "kpod pull from docker without tag" {
 	run ${KPOD_BINARY} $KPOD_OPTIONS pull debian
 	echo "$output"
 	[ "$status" -eq 0 ]
+	run ${KPOD_BINARY} $KPOD_OPTIONS rmi debian
+	[ "$status" -eq 0 ]
 }
 
 @test "kpod pull from a non-docker registry with tag" {
 	run ${KPOD_BINARY} $KPOD_OPTIONS pull registry.fedoraproject.org/fedora:rawhide
 	echo "$output"
 	[ "$status" -eq 0 ]
+	run ${KPOD_BINARY} $KPOD_OPTIONS rmi registry.fedoraproject.org/fedora:rawhide
+	[ "$status" -eq 0 ]
 }
 
 @test "kpod pull from a non-docker registry without tag" {
 	run ${KPOD_BINARY} $KPOD_OPTIONS pull registry.fedoraproject.org/fedora
 	echo "$output"
 	[ "$status" -eq 0 ]
+	run ${KPOD_BINARY} $KPOD_OPTIONS rmi registry.fedoraproject.org/fedora
+	[ "$status" -eq 0 ]
 }
 
 @test "kpod pull using digest" {
-	run ${KPOD_BINARY} $KPOD_OPTIONS pull debian@sha256:7d067f77d2ae5a23fe6920f8fbc2936c4b0d417e9d01b26372561860750815f0
+	run ${KPOD_BINARY} $KPOD_OPTIONS pull alpine@sha256:1072e499f3f655a032e88542330cf75b02e7bdf673278f701d7ba61629ee3ebe
 	echo "$output"
 	[ "$status" -eq 0 ]
+	run ${KPOD_BINARY} $KPOD_OPTIONS rmi alpine:latest
+	[ "$status" -eq 0 ]
 }
 
 @test "kpod pull from a non existent image" {
 	run ${KPOD_BINARY} $KPOD_OPTIONS pull umohnani/get-started
 	echo "$output"
 	[ "$status" -ne 0 ]
 }
+
+@test "kpod history default" {
+	run ${KPOD_BINARY} ${KPOD_OPTIONS} pull $IMAGE
+	[ "$status" -eq 0 ]
+	run ${KPOD_BINARY} ${KPOD_OPTIONS} history $IMAGE
+	echo "$output"
+	[ "$status" -eq 0 ]
+	run ${KPOD_BINARY} $KPOD_OPTIONS rmi $IMAGE
+	[ "$status" -eq 0 ]
+}
+
+@test "kpod history with format" {
+	run ${KPOD_BINARY} ${KPOD_OPTIONS} pull $IMAGE
+	[ "$status" -eq 0 ]
+	run ${KPOD_BINARY} ${KPOD_OPTIONS} history --format "{{.ID}} {{.Created}}" $IMAGE
+	echo "$output"
+	[ "$status" -eq 0 ]
+	run ${KPOD_BINARY} $KPOD_OPTIONS rmi $IMAGE
+	[ "$status" -eq 0 ]
+}
+
+@test "kpod history human flag" {
+	run ${KPOD_BINARY} ${KPOD_OPTIONS} pull $IMAGE
+	[ "$status" -eq 0 ]
+	run ${KPOD_BINARY} ${KPOD_OPTIONS} history --human=false $IMAGE
+	echo "$output"
+	[ "$status" -eq 0 ]
+	run ${KPOD_BINARY} $KPOD_OPTIONS rmi $IMAGE
+	[ "$status" -eq 0 ]
+}
+
+@test "kpod history quiet flag" {
+	run ${KPOD_BINARY} ${KPOD_OPTIONS} pull $IMAGE
+	[ "$status" -eq 0 ]
+	run ${KPOD_BINARY} ${KPOD_OPTIONS} history -q $IMAGE
+	echo "$output"
+	[ "$status" -eq 0 ]
+	run ${KPOD_BINARY} $KPOD_OPTIONS rmi $IMAGE
+	[ "$status" -eq 0 ]
+}
+
+@test "kpod history no-trunc flag" {
+	run ${KPOD_BINARY} ${KPOD_OPTIONS} pull $IMAGE
+	[ "$status" -eq 0 ]
+	run ${KPOD_BINARY} ${KPOD_OPTIONS} history --no-trunc $IMAGE
+	echo "$output"
+	[ "$status" -eq 0 ]
+	run ${KPOD_BINARY} $KPOD_OPTIONS rmi $IMAGE
+	[ "$status" -eq 0 ]
+}
+
+@test "kpod history json flag" {
+	run ${KPOD_BINARY} ${KPOD_OPTIONS} pull $IMAGE
+	run bash -c "${KPOD_BINARY} ${KPOD_OPTIONS} history --json $IMAGE | /usr/bin/json_pp"
```

Comment:
Ok, I will try that. I was looking at the log for Travis and everything passed so I thought it was good. Didn't realise it could fail for atomic bot.